deploy/panorama_command.sh

```diff
@@ -25,7 +25,7 @@ export HEROKU_DEPLOY_PATH=${HEROKU_DEPLOY_PATH:=/usr/local/bin/heroku-deploy}
 # Build docker image
 docker build \
   --build-arg buildkite_agent_uid=$UID \
-  -t heroku-deploy-img \
+  -t ${HEROKU_DEPLOY_IMAGE_NAME:=heroku-deploy-img} \
```

Comment:
This change lets us specify a different Docker image tag via the `HEROKU_DEPLOY_IMAGE_NAME` env var if we want, as explained above. If that var isn't set, which it won't be for normal Buildkite builds, we'll continue to use `heroku-deploy-img` as we have been.